demo/vue.htm

```diff
@@ -1,58 +1,59 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Vue Demo - PayPal JS</title>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Vue Demo - PayPal JS</title>
 
-  <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-core/5.8.38/browser.js"></script>
-  <script src="https://www.paypal.com/sdk/js?client-id=sb"></script>
-</head>
-<body>
+    <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-core/5.8.38/browser.js"></script>
+    <script src="https://www.paypal.com/sdk/js?client-id=sb"></script>
+  </head>
+  <body>
 
-  <div id="container">
-    <app></app>
-  </div>
-  <script type="text/babel">
-  
-    const PayPalButton = paypal.Buttons.driver("vue", window.Vue);
-  
-    Vue.component("app", {
-      template: `
-        <paypal-buttons [props]="{
-            createOrder: createOrder,
-            onApprove: onApprove
-        }"></paypal-buttons>
-      `,
-      components: {
-        "paypal-buttons": PayPalButton,
-      },
-  
-      computed: {
-        createOrder: function (data, actions) {
-          return actions.order.create({
-            purchase_units: [
-              {
-                amount: {
-                  value: "0.01",
-                },
-              },
-            ],
-          });
+    <div id="container">
+      <app></app>
+    </div>
+    <script type="text/babel">
+    
+      const PayPalButton = paypal.Buttons.driver("vue", window.Vue);
+    
+      Vue.component("app", {
+        template: `
+          <paypal-buttons :on-approve="onApprove" :create-order="createOrder" />
+        `,
+        components: {
+          "paypal-buttons": PayPalButton,
         },
-        onApprove: function (data, actions) {
-          return actions.order.capture();
+    
+        computed: {
```

Comment:
This needs to be `methods`

Reply:
@Artem-Schander thanks for the feedback. In this case we used `computed` because we don't pass in any param  with this methods and we need to Vue to keep track of them when down stream the JS SDK call them (This is just a wrapper for the JS SDK).